### PR TITLE
Convert the C# sample projects to PackageReference format

### PR DESCRIPTION
**Pull request by Numpsy**
Part one of the suggestion from #526 - convert the C# sample projects from packages.config to PackageReference format (and delete all the packages.config files)

_I certify that I own, and have sufficient rights to contribute, all source code and related material intended to be compiled or integrated with the source code for the SharpZipLib open source product (the "Contribution"). My Contribution is licensed under the MIT License._

### CODE REVIEW

File: samples/ICSharpCode.SharpZipLib.Samples/cs/Cmd_BZip2/Cmd_BZip2.csproj [hunk 1]
```diff
@@ -103,5 +99,10 @@ copy Cmd_BZip2.exe bunzip2.exe</PostBuildEvent>
   <ItemGroup>
     <Content Include="readme.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="SharpZipLib">
+      <Version>1.3.0</Version>
```
Comment (piksel) [inline samples/ICSharpCode.SharpZipLib.Samples/cs/Cmd_BZip2/Cmd_BZip2.csproj:104]:
```suggestion
      <Version>1.3.1</Version>
```

File: samples/ICSharpCode.SharpZipLib.Samples/cs/Cmd_Checksum/Cmd_Checksum.csproj [hunk 2]
```diff
@@ -102,5 +98,10 @@
   <ItemGroup>
     <Content Include="readme.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="SharpZipLib">
+      <Version>1.3.0</Version>
```
Comment (piksel) [inline samples/ICSharpCode.SharpZipLib.Samples/cs/Cmd_Checksum/Cmd_Checksum.csproj:103]:
```suggestion
      <Version>1.3.1</Version>
```

File: samples/ICSharpCode.SharpZipLib.Samples/cs/Cmd_GZip/Cmd_GZip.csproj [hunk 3]
```diff
@@ -102,5 +98,10 @@ copy Cmd_GZip.exe gunzip.exe</PostBuildEvent>
   <ItemGroup>
     <Content Include="readme.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="SharpZipLib">
+      <Version>1.3.0</Version>
```
Comment (piksel) [inline samples/ICSharpCode.SharpZipLib.Samples/cs/Cmd_GZip/Cmd_GZip.csproj:103]:
```suggestion
      <Version>1.3.1</Version>
```

File: samples/ICSharpCode.SharpZipLib.Samples/cs/Cmd_Tar/Cmd_Tar.csproj [hunk 4]
```diff
@@ -93,6 +89,11 @@
       <Install>true</Install>
     </BootstrapperPackage>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="SharpZipLib">
+      <Version>1.3.0</Version>
```
Comment (piksel) [inline samples/ICSharpCode.SharpZipLib.Samples/cs/Cmd_Tar/Cmd_Tar.csproj:94]:
```suggestion
      <Version>1.3.1</Version>
```

File: samples/ICSharpCode.SharpZipLib.Samples/cs/Cmd_ZipInfo/Cmd_ZipInfo.csproj [hunk 5]
```diff
@@ -101,5 +97,10 @@
   <ItemGroup>
     <Content Include="readme.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="SharpZipLib">
+      <Version>1.3.0</Version>
```
Comment (piksel) [inline samples/ICSharpCode.SharpZipLib.Samples/cs/Cmd_ZipInfo/Cmd_ZipInfo.csproj:102]:
```suggestion
      <Version>1.3.1</Version>
```

File: samples/ICSharpCode.SharpZipLib.Samples/cs/CreateZipFile/CreateZipFile.csproj [hunk 6]
```diff
@@ -109,5 +105,10 @@
   <ItemGroup>
     <Content Include="readme.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="SharpZipLib">
+      <Version>1.3.0</Version>
```
Comment (piksel) [inline samples/ICSharpCode.SharpZipLib.Samples/cs/CreateZipFile/CreateZipFile.csproj:110]:
```suggestion
      <Version>1.3.1</Version>
```

File: samples/ICSharpCode.SharpZipLib.Samples/cs/FastZip/FastZip.csproj [hunk 7]
```diff
@@ -89,5 +88,10 @@
       <Install>true</Install>
     </BootstrapperPackage>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="SharpZipLib">
+      <Version>1.3.0</Version>
```
Comment (piksel) [inline samples/ICSharpCode.SharpZipLib.Samples/cs/FastZip/FastZip.csproj:93]:
```suggestion
      <Version>1.3.1</Version>
```

File: samples/ICSharpCode.SharpZipLib.Samples/cs/sz/sz.csproj [hunk 8]
```diff
@@ -87,5 +83,10 @@
       <Install>true</Install>
     </BootstrapperPackage>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="SharpZipLib">
+      <Version>1.3.0</Version>
```
Comment (piksel) [inline samples/ICSharpCode.SharpZipLib.Samples/cs/sz/sz.csproj:88]:
```suggestion
      <Version>1.3.1</Version>
```

File: samples/ICSharpCode.SharpZipLib.Samples/cs/unzipfile/unzipfile.csproj [hunk 9]
```diff
@@ -58,11 +55,15 @@
...
   <ItemGroup>
     <None Include="App.config" />
-    <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
     <Content Include="readme.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="SharpZipLib">
+      <Version>1.3.0</Version>
```
Comment (piksel) [inline samples/ICSharpCode.SharpZipLib.Samples/cs/unzipfile/unzipfile.csproj:64]:
```suggestion
      <Version>1.3.1</Version>
```

File: samples/ICSharpCode.SharpZipLib.Samples/cs/viewzipfile/viewzipfile.csproj [hunk 10]
```diff
@@ -58,11 +55,15 @@
...
   <ItemGroup>
     <None Include="App.config" />
-    <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
     <Content Include="readme.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="SharpZipLib">
+      <Version>1.3.0</Version>
```
Comment (piksel) [inline samples/ICSharpCode.SharpZipLib.Samples/cs/viewzipfile/viewzipfile.csproj:64]:
```suggestion
      <Version>1.3.1</Version>
```

File: samples/ICSharpCode.SharpZipLib.Samples/cs/zf/zf.csproj [hunk 11]
```diff
@@ -87,5 +83,10 @@
       <Install>true</Install>
     </BootstrapperPackage>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="SharpZipLib">
+      <Version>1.3.0</Version>
```
Comment (piksel) [inline samples/ICSharpCode.SharpZipLib.Samples/cs/zf/zf.csproj:88]:
```suggestion
      <Version>1.3.1</Version>
```

**Review by piksel — APPROVED** [review]
LGTM. Added suggestions for updating to the just released v1.3.1

**Comment by Numpsy** [comment]
It should also be possible to put the SharpZipLib package version and/or the PackageReference in a shared .props file (such that the version can be updated in once place rather than having to change every project file). Not sure if that's a good idea or not though.

**Pull request merged**